app/javascript/article-form/elements/bodyPreview.jsx

```diff
@@ -29,15 +29,12 @@ function titleArea(previewResponse, version, articleState) {
     });
   }
 
-  let coverImage = '';
+  let coverImage = articleState.mainImage || '';
```

Comment:
So the main issue is the shape of `articleState` between `v1` and `v2`? `v1` has no image on article state, just `cover_image` on the `previewResponse`? And `v2` has `mainImage` on `articleState` only correct?

Reply:
Right; the main issue here was that we were always relying on `cover_image` to pull the image in preview mode, but that only applies to the v1 editor.

Reply:
@vaidehijoshi it'd be awesome if we could add this note as a comment in the file :) because I'm sure we'll forget at some point hahaha

Reply:
good call! 👍